submissions/temitopeJohnson/conditionals/index.js

```diff
@@ -0,0 +1,29 @@
+var x,
```

Comment:
your variables are not explanatory on its own, another developer should be  able to read and understand what you have done and the solution you are trying to solve.